gradle-language-server/src/main/java/com/microsoft/gradle/resolver/GradleLibraryResolver.java

```diff
@@ -190,4 +208,33 @@ private void getGradleLibraries(Path jarPath, JarFile jarFile) {
       }
     }
   }
+
+  private void resolveJavaConfigurations() {
+    JavaClass javaPluginClass = this.gradleLibraries.get(GradleLibraryResolver.JAVA_PLUGIN);
+    if (javaPluginClass == null) {
+      return;
+    }
+    for (Field field : javaPluginClass.getFields()) {
+      if (field.getName().endsWith("CONFIGURATION_NAME")) {
```

Comment:
See https://github.com/gradle/gradle/blob/499dc4d71d716c84187135a49e762dea35c8caab/subprojects/plugins/src/main/java/org/gradle/api/plugins/JavaPlugin.java#L122-L244 for the reference